blog/_posts/2022-03-30-tilt-news-march-2022.md

```diff
@@ -0,0 +1,56 @@
+---
+slug: "tilt-news-march-2022"
+date: 2022-03-30
+author: lian
+layout: blog
+title: "Tilt News, March 2022"
+image: "/assets/images/tilt-news-march-2022/pexels-green-windmills.jpg"
+image_caption: "A dutch windmill and house all in green. Photo by Helena Jankovičová Kováčová from <a href='https://www.pexels.com/photo/a-green-windmill-near-the-house-6470648/'>Pexels</a>"
+description: "Updated posts and cool new stuff we found during spring-cleaning 🌼"
+tags:
+  - news
+---
+
+你好 Tilt 用户,
+
+This month, vacation season started at Tilt, so we’re going to have a very brief update.
+
+## Updates from the ecosystem
+Developer tooling is gaining more and more attention from engineers and investors alike. While that means tougher competition, in our opinion, this will also improve the quality of the overall ecosystem.  
+So, we’re excited to congratulate [Okteto on their Series A](https://www.okteto.com/blog/a-15m-series-a-to-disrupt-modern-development/) and [Signadot](https://www.prnewswire.com/news-releases/y-combinator-alum-signadot-raises-4m-led-by-redpoint-ventures-to-reimagine-microservices-testing-on-kubernetes-301488164.html) on their Seed funding!
+
+## Optimize your Tilt experience
+
+### ...running containers locally
+Many of our users have been experimenting with all kinds of local container orchestration tools to suit their individual needs. Since we last wrote about Rancher Desktop, a lot has changed. In this blog post, Milas gives us a [rundown of different container runtimes with Rancher Desktop](https://blog.tilt.dev/2022/03/04/rancher-desktop-container-runtimes.html).
+
+### ...sharing Tiltfile best practices
+Our vision for Tilt is to make developers’ lives easier. But we don’t stop at individual engineers. Because we believe that sharing is caring, Nick has written this blog post to explain how you can share your custom Tilt magic in a [private extensions library](https://blog.tilt.dev/2022/03/01/dev-env-libraries.html).
+
+### ...running subsets of resources
+Just a couple of weeks ago, we released the [Disable Resources Feature](https://docs.tilt.dev/disable_resources.html). With that in place, teams and individual devs can now [create a catalog of resources](https://blog.tilt.dev/2022/03/03/resource-catalog.html), tailored to their specific requirements and workflows. Spinning up only the resources you need, while still being able to bring up disabled resources with just one click is now a breeze.
+
+### ...managing Pulumi resources
```

Comment:
"manage" isn't really how i would frame this. It sounds WAY too much like what Pulumi already does. (the 3 pillars on their marketing site are literally "Build, Deploy, Manage")

maybe "attach live updates to Pulumi resources"? "Define a dev environment with Pulumi scripts"?

Reply:
I like the last one 👍